openrouteservice-api-tests/src/test/java/org/heigit/ors/services/routing/ParamsTest.java

```diff
@@ -588,8 +586,7 @@ public void expectCarToRejectWalkingAvoidables() {
 				.param("geometry", "true")
 				.param("profile", getParameter("carProfile"))
 				.param("options", options.toString())
-				.when()
-				.log().all()
+				.when().log().all()
```

Comment:
I got rid of all the then().log().all(), not the when().log().all(), since these tend to be longer, I can remove them entirely since the tests don't depend on them or anything.

Reply:
No worries, I was just wondering why did you change them.